README.md

```diff
@@ -0,0 +1,50 @@
+# @artsy/icons
+
+[@artsy/icons](https://github.com/artsy/icons) is the canonical source for icons used at Artsy, serving as an automatic build pipeline for generating JSX representations of .svg sources that can easily be imported into React apps.
+
+To view available icons, see [the docs](https://icons.artsy.net).
+
+## Meta
+
+- **State:** production
+- **GitHub:** [https://github.com/artsy/icons](https://github.com/artsy/icons)
+- **CI/Deploys:** [CircleCi](https://circleci.com/gh/artsy/force); merged PRs to `artsy/icons#main` are automatically deployed to NPM
+- **Docs:** [https://force-storybook.artsy.net](https://force-storybook.artsy.net)
+- **Point People:** [@dzucconi](https://github.com/dzucconi)
+
+[![Build Status](https://circleci.com/gh/artsy/icons.svg?style=svg)](https://circleci.com/gh/artsy/icons)
+
+## Installation and Usage
+
+```bash
+yarn add @artsy/icons
+```
+
+And then later, import icons like so:
+
+```tsx
+import { ArtsyLogo } from "@artsy/icons"
```

Comment:
Actually we don't want to import like this. I should probably just disable this and figure out a slightly different way of importing an index for the docs site. (This will wind up importing ALL the icons like Palette currently does)

Reply:
What we want is to force people to go to icons.artsy.net and click-to-copy the imports there. Which will import only that icon.

Reply:
Oh hmmmm 👀